spec.html

```diff
@@ -6284,7 +6284,7 @@
     <p>An agent becomes <em>blocked</em> when its running execution context waits synchronously and indefinitely for an external event.  Only agents whose Agent Record's [[CanBlock]] property is ~true~ can become blocked in this sense.  An <em>unblocked</em> agent is one that is not blocked. </p>
 
     <emu-note>
-      <p> The shared memory proposal introduces a memory for blocking in the Atomics.wait method. </p>
+      <p> The shared memory proposal introduces a method for blocking in the Atomics.wait method. </p>
```

Comment:
Stylistically it's still a disaster, of course ("method ... method").  Sigh.  Maybe a "means of blocking in the A.w method" would be better.